src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1211,6 +1211,54 @@ key is not present, no replacement occurs):
 </task>
 ----
 
+== Fork environment
+A fork execution environment is a new java virtual machine which is started exclusively to execute a task. Starting a
+new java virtual machine means that the task inside it will run in a new environment. This environment can
+be set up by the creator of the task. A new java virtual machines is set up with a new classpath,
+new system properties and more customization.
+
+=== Docker fork execution environment
+A Docker fork execution environment executes a java virtual machine inside a Docker container. Hence the task
+which is executes in the Docker fork execution environment can be customized by the java virtual machine settings
+and by the tool-set and isolation provided by Docker containers.
+
+TIP: If your task needs to install new software or updates software, then run it inside a Docker container.
+That way other tasks will not be affected by changes, because Docker containers provide isolation so that
+the host machine's software stays the same.
+
+==== How to use a fork execution Environment
+Select Docker in the ‘Fork Execution Environment’ Dropdown:
+
+* A sample Docker Environment Setting will appear.
```

Comment:
I wanted to break the text visually, to make clear that is is a result of the above action. But it is inconsistent, I removed it.